src/FSharpLint.Core/Rules/Conventions/Naming/NamingHelper.fs

```diff
@@ -361,3 +361,22 @@ let rec identFromSimplePat = function
     | SynSimplePat.Id(ident, _, _, _, _, _) -> Some(ident)
     | SynSimplePat.Typed(p, _, _) -> identFromSimplePat p
     | SynSimplePat.Attrib(_) -> None
+
+let rec isNested args nodeIndex =
+    let parent = args.SyntaxArray.[nodeIndex].ParentIndex
+    let actual = args.SyntaxArray.[parent].Actual
+
+    match actual with
+    | AstNode.Expression (SynExpr.LetOrUse (_, _, _, _, _)) -> true
+    | _ -> false
+
+let getFunctionIdents _ =
+    function
```

Comment:
I'm not a fan of using the keyword `function` here, can you just write a normal function?

Reply:
Yes, and better to rename `_` to `accessibility` or `_accessibility` just to see what is being ignored